app/assets/javascripts/components/student_dashboard/components/lesson_card.es6.jsx

```diff
@@ -0,0 +1,46 @@
+class LessonCard extends React.Component{
+
+
+  render() {
+    return (
+      <div className="lesson-card">
```

Comment:
if you don't have any styling under lesson-card, why do you need nest a div in a div (card-container in lesson-card)